nncf/quantization/algorithms/weight_compression/openvino_backend.py

```diff
@@ -153,19 +153,22 @@ def transform_model(
             compressed_const = opset.constant(
                 compressed_weight.tensor.data, dtype=compression_dtype, name=const_node_name
             )
-            converted_const = opset.convert(compressed_const, const_dtype)
+            converted_const = opset.convert(compressed_const, ov.Type.f32)
```

Comment:
please, check your solution on the model with FP16 weights and activations.

Reply:
Could you share compression subgraph for model with FP16 weights and activations?

Reply:
Thanks, I've added the logic to correctly process the case of FP16 activations.